dcat/rdf/dcat.ttl

```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
   rdfs:comment """Μπορεί να είναι οποιουδήποτε είδους URL που δίνει πρόσβαση στη διανομή ενός συνόλου δεδομένων. Π.χ. ιστοσελίδα αρχικής πρόσβασης,
-          μεταφόρτωση, feed URL, σημείο διάθεσης SPARQL. Να χρησιμοποιείται όταν ο κατάλογος δεν περιέχει πληροφορίες εαν πρόκειται ή όχι για μεταφορτώσιμο αρχείο."""@el ;
-  rdfs:comment "أي رابط يتيح الوصول إلى البيانات. إذا كان الرابط هو ربط مباشر لملف يمكن تحميله استخدم الخاصية downloadURL"@ar ;
-  rdfs:comment "データセットの配信にアクセス権を与えるランディング・ページ、フィード、SPARQLエンドポイント、その他の種類の資源。"@ja ;
-  rdfs:domain dcat:Distribution ;
-  rdfs:isDefinedBy <http://www.w3.org/TR/vocab-dcat/> ;
-  rdfs:label "URL d'accès"@fr ;
-  rdfs:label "URL de acceso"@es ;
-  rdfs:label "indirizzo di accesso"@it ;
-  rdfs:label "URL πρόσβασης"@el ;
-  rdfs:label "access address"@en ;
-  rdfs:label "přístupová adresa"@cs ;
-  rdfs:label "رابط وصول"@ar ;
-  rdfs:label "アクセスURL"@ja ;
-  rdfs:range rdfs:Resource ;
-  owl:propertyChainAxiom (
-      dcat:accessService
-      dcat:endpointURL
-    ) ;
-  skos:editorialNote "Status: English Definition text modified by DCAT revision team, updated Italian and Czech translation provided, translations for other languages pending."@en ;
-  skos:editorialNote "rdfs:label, rdfs:comment and skos:scopeNote have been modified. Non-english versions except for Italian must be updated."@en ;
+          μεταφόρτωση, feed URL, σημείο διάθεσης SPARQL. Να χρησιμοποιείται όταν ο κατάλογος δεν περιέχει πληροφορίες εαν πρόκειται ή όχι για μεταφορτώσιμο αρχείο."""@el ;
+  rdfs:comment "أي رابط يتيح الوصول إلى البيانات. إذا كان الرابط هو ربط مباشر لملف يمكن تحميله استخدم الخاصية downloadURL"@ar ;
+  rdfs:comment "データセットの配信にアクセス権を与えるランディング・ページ、フィード、SPARQLエンドポイント、その他の種類の資源。"@ja ;
+  rdfs:domain dcat:Distribution ;
+  rdfs:isDefinedBy <http://www.w3.org/TR/vocab-dcat/> ;
+  rdfs:label "URL d'accès"@fr ;
+  rdfs:label "URL de acceso"@es ;
+  rdfs:label "URL πρόσβασης"@el ;
+  rdfs:label "access address"@en ;
+  rdfs:label "indirizzo di accesso"@it ;
+  rdfs:label "přístupová adresa"@cs ;
+  rdfs:label "رابط وصول"@ar ;
+  rdfs:label "アクセスURL"@ja ;
+  rdfs:range rdfs:Resource ;
+  owl:propertyChainAxiom (
+      dcat:accessService
+      dcat:endpointURL
+    ) ;
+  skos:definition """A URL of a resource that gives access to a distribution of the dataset. E.g. landing page, feed, SPARQL endpoint.
+          Use for all cases except a simple download link, in which case downloadURL is preferred."""@en ;
+  skos:definition """Ceci peut être tout type d'URL qui donne accès à une distribution du jeu de données. Par exemple, un lien à une page HTML contenant un lien au jeu de données,
+					un Flux RSS, un point d'accès SPARQL. Utilisez le lorsque votre catalogue ne contient pas d'information sur quoi il est ou quand ce n'est pas téléchargeable."""@fr ;
+  skos:definiti
```

Comment:
To stay coherent with the rest of the dcat,  I suspect we should add rdfs:comments for these skos:definition.

Reply:
Not necessarily. We should retain `rdfs:comment` where it was used in DCAT 2014. But if we prefer `skos:definition` moving forward, then we don't necessarily need to duplicate into `rdfs:comment` for new elements.